### PR TITLE
alazar_seq_mode_to_bool

### PR DESCRIPTION
**Pull request by nataliejpg**
WYSIWYG

TODO: 
- [x] test

### CODE REVIEW

File: qdev_wrappers/customised_instruments/AlazarTech_ATS9360_ext.py [hunk 1]
```diff
@@ -8,24 +8,30 @@ def __init__(self, name):
         self.add_parameter(name='seq_mode',
                            get_cmd=self._get_seq_mod,
                            set_cmd=self._set_seq_mode,
-                           vals=vals.Enum('on', 'off'))
+                           vals=vals.Enum('on', 'off'),
+                           get_cmd=get_parser_on_off,
+                           set_cmd=set_parser_on_off)
```
Comment (jenshnielsen) [inline qdev_wrappers/customised_instruments/AlazarTech_ATS9360_ext.py]: I can't seem to spot that this is imported anywhere?

Comment (nataliejpg) [reply qdev_wrappers/customised_instruments/AlazarTech_ATS9360_ext.py]: you're right, let me fix that

**Comment by nataliejpg** [comment]
have tested and it works, I realise this is slightly api changing (in terms of returning a bool instead of a string) but if I make a shout out on the qcodes announcements slack can I merge this?

**Comment by jenshnielsen** [comment]
Yes that's probably fine. @ThorvaldLarsen?

**Comment by ThorvaldLarsen** [comment]
Looks good to me. 

@jenshnielsen is there any chance that there will be a qcodes parser for Boolean parameters? Almost all instruments has at least one such parameter (e.g. output on/off) and would be nice to have a common behavior for all instruments.

**Comment by ThorvaldLarsen** [comment]
Yes, guess I knew that there was a issue with it... 
However, looking at it seems everyone agrees and it is just a question of where the PR goes to main qcodes. It could live next to validators but maybe as a separate file in utils.

**Comment by nataliejpg** [comment]
I'll merge this now, @ThorvaldLarsen and @jenshnielsen maybe continue the on/off discussion in the issue I mentioned?

**Comment by jenshnielsen** [comment]
Yes I think we should just bring this thing into qcodes as is

**Pull request merged**